extensions/ARB/ARB_gl_spirv.txt

```diff
@@ -620,7 +620,8 @@ Modifications to Chapter 7 of the OpenGL 4.5 (Core Profile) Specification
     User-defined interface variables must be decorated with a *Location*
     and can also be decorated with a *Component*. These correspond to the
     location and component discussed in section 7.4.1 "Shader Interface
-    Matching".
+    Matching". Uniform and shader storage block variables must also be
```

Comment:
Does this include other resources too, like textures, samplers, atomic counters, etc.?  Or, they are covered elsewhere?